customer_pricing_in_product_view/models/res_partner.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Use short headers like the one in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py